helm/reporting-service/templates/deployment.yaml

```diff
@@ -37,16 +37,30 @@ spec:
               containerPort: 80
               protocol: TCP
           env:
-            - name: "PORT"
+            {{- if .Values.auth }}
+            - name: ORY_KETO_READ_URL
+              value: '{{ include "common.backends.keto.readURL" . }}'
+            {{- end }}
+            - name: WATCH_RESOURCE_GROUP
+              value: "{{ .Values.operator.resourceGroup }}"
+            - name: WATCH_RESOURCE_VERSION
+              value: '{{ .Values.operator.resourceVersion }}'
```

Comment:
Should resource version actually be in env, or configurable at all? Does the operator code handle a certain version of the resource? If so, should this be fixed, non-variable?